lib/http_server.js

```diff
@@ -20,6 +20,7 @@ var deviceFormatter = require('./api_formats/siren/device.siren');
 var rels = require('zetta-rels');
 
 var querytopic = require('./query_topic');
+console.log(url);
```

Comment:
yeah sorry for that, just saw it too while executing the tests and was wonder where the heck it came from :)